Formula/helmfile.rb

```diff
@@ -34,7 +34,8 @@ def install
     - name: test
     EOS
     system Formula["kubernetes-helm"].opt_bin/"helm", "init", "--client-only"
-    output = '"stable" has been added to your repositories'
-    assert_match output, shell_output("#{bin}/helmfile -f helmfile.yaml repos")
+    system "#{bin}/helmfile -f helmfile.yaml repos"
+    output = "helmfile version"
+    assert_match output, shell_output("#{bin}/helmfile -v")
```

Comment:
I spoke too soon. This test is still a downgrade/simplification. Does `helmfile -f helmfile.yaml repos` no longer print anything to stdout? If it does can we check the output? Or look for a log file or some other file created by this command?

Reply:
ping @chenrui333

Reply:
ack

Reply:
Seems like there is not very easy way other just check the version output. 

How can I assert match the following output?

```
==> /usr/local/Cellar/helmfile/0.64.2/bin/helmfile -f helmfile.yaml repos
Adding repo stable https://kubernetes-charts.storage.googleapis.com/
"stable" has been added to your repositories

Updating repo
Hang tight while we grab the latest from your chart repositories...
...Skip local chart repository
...Successfully got an update from the "stable" chart repository
Update Complete.
```

Reply:
It seems always show me the empty string during the assert_match test.